features/issue.feature

```diff
@@ -31,6 +31,66 @@ Feature: hub issue
       """
     And the exit status should be 0
 
+  Scenario: Fetch issues not assigned to any milestone
+    Given the GitHub API server:
+    """
+    get('/repos/github/hub/issues') {
+      assert :milestone => "none"
+
+      json []
+    }
+    """
+    When I run `hub issue -M none`
+    Then the exit status should be 0
```

Comment:
Instead of checking if exit status was 0, the previous step could be "When **successfully** I run"